site/en/develop/expression.md

```diff
@@ -0,0 +1,109 @@
+---
+id: expression.md
+title: Predicate Expressions
+---
+
+# Predicate Expressions
+A predicate is an expression that evaluates to a boolean value. Milvus conduct scalar filtering by searching with predicates. A predicate expression, when evaluated, returns either TRUE or FALSE.
+
+View <a href="https://milvus.io/api-reference/pymilvus-orm/v2.0.0rc1/api/collection.html">Python SDK API Reference</a> for information about how to use predicate expressions.
```

Comment:
"/api-reference/pymilvus-orm/{{var.milvus_python_orm_sdk_version}}/install.html"


# We use internal link while linking within Milvus.io. And we use placeholders to represent variables that may change among versions.

Reply:
Besides, the contributing guide for Milvus-doc will be in position soon. This will be covered for sure. 😄